packages/core/types/@chainlink/index.d.ts

```diff
@@ -102,6 +102,18 @@ declare module '@chainlink/types' {
     warning?: string
   }
 
+  export type WSSubscriptionHandler = {
+    connection: {
+      url: string | any
+      protocol?: any
```

Comment:
We should clean up the typing in this file, LMK if you need help doing so!

Reply:
We should yep, I'll open an issue